practice/kimdowon.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
```suggestion
    <title>의미 있는 제목</title>
```

의미 있는 제목으로 바꿔 주시면 좋을 것 같습니다!